spec/models/case_study_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe CaseStudy do
+  let(:content_item) do
```

Comment:
To be consistent with other tests this should be called `content_store_response`.